drivers/stmpe811/stmpe811.c

```diff
@@ -194,6 +194,15 @@ int stmpe811_read_touch_position(stmpe811_t *dev, stmpe811_touch_position_t *pos
     /* Acquire I2C device */
     i2c_acquire(STMPE811_DEV_I2C);
 
+    /* we don't use the fifo yet, discard it and get the next event */
+    _reset_fifo(dev);
```

Comment:
I tried the last version of this PR with #14188 and it doesn't work well. If I move this line to line 269 (just before `*state = STMPE811_TOUCH_STATE_RELEASED;`, then it works like a charm in both the driver test application and #14188. Would mind applying this change @benpicco ?

Reply:
I allowed myself to push the update once again in your branch. I hope this is fine ;)